app/assets/stylesheets/landing.scss

```diff
@@ -64,6 +68,45 @@
 
 }
 
+.footer-section {
+  align-items: center;
+  background-color: $orange;
+  box-shadow: 0 -8px 20px 0 #11111129;
```

Comment:
`#11111129` this is not supported in all browsers, use `rgba(r,g,b,a);` instead.